spec/services/partner_fetch_requestable_items_service_spec.rb

```diff
@@ -4,8 +4,24 @@
   describe '#call' do
     subject { described_class.new(partner_id: partner_id).call }
     let(:partner_id) { partner.id }
-    let(:partner) { create(:partner) }
-    let(:organization) { partner.organization }
+    let(:partner) { create(:partner, organization:) }
+    let(:organization) { create(:organization, skip_items: true, items:) }
```

Comment:
wow... I see why Rubocop enforced this but I really don't like it. `items` isn't a variable here. :( Can we rename the `let` block so it's not so "magic"? Same for `organization`